src/PrestaShopBundle/Bridge/AdminController/LegacyControllerBridge.php

```diff
@@ -128,7 +236,7 @@ public function setMedia($isNewTheme = false): void
         ]);
 
         Media::addJsDef([
-            'changeFormLanguageUrl' => $this->getContext()->link->getAdminLink(
+            'changeFormLanguageUrl' => Context::getContext()->link->getAdminLink(
```

Comment:
```suggestion
            'changeFormLanguageUrl' => $this->getContext()->link->getAdminLink(
```

Reply:
The interest of injecting the LegacyContext is not to rely on the static `Context::getContext()` anymore, so they should all be replaced